app/models/sign_off.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: sign_offs
+#
+#  id                :bigint           not null, primary key
+#  sign_off_type     :string           not null
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#  academic_cycle_id :bigint           not null
+#  provider_id       :bigint           not null
+#  user_id           :bigint           not null
+#
+# Indexes
+#
+#  index_sign_offs_on_academic_cycle_id                  (academic_cycle_id)
+#  index_sign_offs_on_provider_id                        (provider_id)
+#  index_sign_offs_on_provider_id_and_academic_cycle_id  (provider_id,academic_cycle_id) UNIQUE
+#  index_sign_offs_on_user_id                            (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (academic_cycle_id => academic_cycles.id)
+#  fk_rails_...  (provider_id => providers.id)
+#  fk_rails_...  (user_id => users.id)
+#
+class SignOff < ApplicationRecord
+  belongs_to :provider
+  belongs_to :academic_cycle
+  belongs_to :user
```

Comment:
I see the audits as primarily being a way to track changes, and the current expectation with sign-offs is that they are created once and not changed after that. We may also in the future want to display a list of a Provider's sign-offs.
So, I think it would be preferable to keep the association with the user, and probably no need to have this auditable at the moment.
If the expectations around sign-offs change then we should review this though.